### DISCUSSION TITLE
[Question]: how to use [[links]] in "tag"s field

### DISCUSSION THREAD

**Discussion by nousernameavailableanymore** [RafaelGB/obsidian-db-folder#511] [Q&A]
### What happened?

I have a tags column for meetings to select the participants.
Instead of just adding names, I would like to add internal links to their pages in the wiki format, like [[participant]].
However, when I try to add such a link, the db folder note will not open anymore (stays blank, no error).

What is the intended way of using internal wiki-links in tags columns?

### What platform were you using?

Desktop

### Version of the plugin

2.7.1

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
currently its not possible. To render markdown links we need a specific way to do it that its not compatible with render another React components, just plain text. Thats why markdown its just avaliable with text and formula types

**Reply by nousernameavailableanymore** [reply]
Hmmm thats unfortunate as it seems (at least for me) to be quite the important feature.
Is there any kind of workaround or any other way to get this to function in a similar way (where a picked value results in an internal link)?

**Reply by RafaelGB** [reply]
Surely over time the API will evolve and allow us that functionality.

**Reply by asterox** [reply]
Hello. Just started with your plugin (well done by the way!) - has there been any progress on this front? Thank you!

**Reply by RafaelGB** [reply]
Hello @asterox ,

In fact, https://github.com/RafaelGB/obsidian-db-folder/issues/613 will be added with the next version =)


This month the plugin will be more stopped, surely it will be included at the beginning of the next

**Reply by asterox** [reply]
Oh that’s great. Thanks very much for the reply!

**Comment by RafaelGB** [comment]
moved to discussions
